tests/lax_numpy_test.py

```diff
@@ -2325,6 +2325,36 @@ def testHistogram(self, shape, dtype, bins, density, weights):
                               tol=tol)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_{}_bins={}_weights={}_density={}".format(
+      jtu.format_shape_dtype_string(shape, dtype), bins, weights, density),
+      "shape": shape,
+      "dtype": dtype,
+      "bins": bins,
+      "weights": weights,
+      "density": density
+    }
+    for shape in [(5,), (12,)]
+    for dtype in int_dtypes
+    for bins in [[2, 2], [[0, 1, 3, 5], [0, 2, 3, 4, 6]]]
```

Comment:
I'd also test `bins=2` to cover the case of passing an integer.

Reply:
Good catch, histogrammdd wasn't handling this case correctly.  The way it should work is if bins is an integer then `bins=D*[bins]`.